packages/definitions-parser/src/lib/definition-parser.ts

```diff
@@ -193,9 +198,20 @@ async function combineDataForAllTypesVersions(
   typingsPackageName: string,
   ls: readonly string[],
   fs: FS,
-  directoryVersion: DirectoryParsedTypingVersion | undefined
+  directoryVersion: DirectoryParsedTypingVersion | undefined,
+  moduleResolutionHost: ts.ModuleResolutionHost,
 ): Promise<Omit<TypingsDataRaw, "libraryVersionDirectoryName">> {
   const { remainingLs, typesVersions, hasPackageJson } = getTypesVersionsAndPackageJson(ls);
+  const packageJson = hasPackageJson
```

Comment:
why did this move? It doesn't look like it was forced by the compiler, so my next guess is readability.

Reply:
I think this is a side effect of an approach I ended up scrapping.